daemons/dss-notify-v2/app.py

```diff
@@ -123,5 +126,24 @@ def _func(subscription):
     with ThreadPoolExecutor(max_workers=20) as e:
         e.map(_func, get_subscriptions_for_replica(replica))
 
+@app.scheduled_function("rate(5 minutes)")
```

Comment:
I *think* this creates another lambda, it might be worthwhile to move it into its own daemon, so we can exert a little more control over it, without effecting changes to the notify_v2.

Reply:
Ok. I was hoping to avoid another daemon, but event journaling probably deserves its own process that we can monitor and throttle as necessary.